first-party/jib-quarkus-extension-gradle/src/main/java/com/google/cloud/tools/jib/gradle/extension/quarkus/JibQuarkusExtension.java

```diff
@@ -172,7 +173,10 @@ private void addDependencyLayer(
 
   private void readJibConfigurations(Project project) {
     JibExtension jibPlugin = project.getExtensions().findByType(JibExtension.class);
-    appRoot = AbsoluteUnixPath.get(jibPlugin.getContainer().getAppRoot());
+    String appRootValue = jibPlugin.getContainer().getAppRoot();
```

Comment:
Should we have written in a default appRoot into the buildPlan? Or do expect the extension to define the approot?

Reply:
The Build Plan and Jib Core don't have an "app root" notion. It's from the build plugins, where the default is `/app`. The extension also [sets `/app` as the default](https://github.com/GoogleContainerTools/jib-extensions/blob/fadb25d1e92a968e967dff015c3cd0361f7a7c88/first-party/jib-quarkus-extension-gradle/src/main/java/com/google/cloud/tools/jib/gradle/extension/quarkus/JibQuarkusExtension.java#L50) accordingly. (It would be nice if extension could get the default (or effective) value from the build plugins, but I think that's not possible unless the extension API changes to pass config values of the build plugins.)